test/e2e/pkg/infra/digitalocean/digitalocean.go

```diff
@@ -24,33 +25,63 @@ func (p *Provider) Setup() error {
 	return nil
 }
 
-const ymlSystemd = "systemd-action.yml"
+var ymlPlaybookSeq int
+
+func getNextPlaybookFilename() string {
+	const ymlPlaybookAction = "playbook-action"
+	ymlPlaybookSeq++
+	return ymlPlaybookAction + strconv.Itoa(ymlPlaybookSeq) + ".yml"
+}
```

Comment:
Is this so that playbooks don't clash with one another?

Reply:
It's for troubleshooting, a late addition to this PR. I wanted to see what _actual_ playbook had been generated, but the runner was overwriting the file with new playbooks.
This way we keep the history, and can inspect.
All these files are part of the "testnet" dir, so they will be wiped out on the next run (or upon cleanup). But, if we see that the number of files is very large for a run (not the case now), we could extend this to only produce, e.g., N files and use them in round-robin.